WBooks/src/app/components/IconTab/constants/iconsName.js

```diff
@@ -0,0 +1,26 @@
+import addNewActive from '../../../../assets/tool_bar/ic_add_new_active.png';
+import addNewPassive from '../../../../assets/tool_bar/ic_add_new_passive.png';
+import libraryActive from '../../../../assets/tool_bar/ic_library_active.png';
+import libraryPassive from '../../../../assets/tool_bar/ic_library_passive.png';
+import myRentalsActive from '../../../../assets/tool_bar/ic_myrentals_active.png';
+import myRentalsPassive from '../../../../assets/tool_bar/ic_myrentals_passive.png';
+import settingsActive from '../../../../assets/tool_bar/ic_settings_active.png';
+import settingsPassive from '../../../../assets/tool_bar/ic_settings_passive.png';
+import wishlistActive from '../../../../assets/tool_bar/ic_wishlist_active.png';
+import wishlistPassive from '../../../../assets/tool_bar/ic_wishlist_passive.png';
```

Comment:
I recommend you use aliases on imports. @santinucera can u teach him to use them?